app/models/users/setting.rb

```diff
@@ -2,6 +2,8 @@ module Users
   class Setting < ApplicationRecord
     self.table_name_prefix = "users_"
 
+    HEX_COLOR_REGEXP = /\A#?(?:\h{6}|\h{3})\z/.freeze
```

Comment:
We still had this on the `ProfileValidator` class but it should be here instead.